conf/application.conf

```diff
@@ -193,3 +201,13 @@ des {
     # It is possible BPR entries will list these unmatched country codes
     non-iso-country-codes = ["AN", "EU", "FC", "NT", "OR", "TP", "UN"]
 }
+
+
+
+tax-enrolment-retry {
+ min-backoff = 3.seconds
+ max-backoff = 20.seconds
+ max-elapsed-time = 60.seconds
```

Comment:
I'd make these the values look a bit more realistic. If you want smaller values for tests then you can inject the config in to allow you to do that